oap-server/server-configuration/configuration-nacos/pom.xml

```diff
@@ -45,4 +45,143 @@
             <artifactId>nacos-client</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
```

Comment:
If you add this, I think you will burn the Travis, isn't it?

Reply:
Just to test Jenkins for now, and the travis isn't necessarily always failed, now it's moved into `CI-with-IT` profile